WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -57,8 +57,7 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 					Wallet.TransactionProcessor,
 					nameof(Wallet.TransactionProcessor.WalletRelevantTransactionProcessed))
 				.Select(_ => Unit.Default)
-				.Merge(Observable.FromEventPattern(Wallet, nameof(Wallet.NewFilterProcessed))
-						.Select(_ => Unit.Default))
+				.Merge(Observable.FromEventPattern(Wallet, nameof(Wallet.NewFilterProcessed)).Throttle(TimeSpan.FromSeconds(1)).Select(_ => Unit.Default))
```

Comment:
> Intuitively it feels enough just to listen to WalletRelevantTransactionProcessed and therefore to remove completely listening to this event there instead of throttling

I think you are right... I would rather remove it. Can you test it like it without that line?

Reply:
I didnt manage to find an issue when deleting this line.
I don't know if that's use by any component but I dont think it is.

Reply:
Don't deal with it, I merge the PR. We will refactor the logic either way. https://github.com/zkSNACKs/WalletWasabi/pull/9137#pullrequestreview-1111791980